velox/exec/WindowFunction.h

```diff
@@ -88,6 +92,7 @@ class WindowFunction {
       const BufferPtr& peerGroupEnds,
       const BufferPtr& frameStarts,
       const BufferPtr& frameEnds,
+      const SelectivityVector& validFrames,
```

Comment:
validFrames -> rows to match the rest of the codebase; the function doesn't need to know why some rows are not requested.